### ISSUE TITLE
This only works for Google Authenticator with my GMail account.

### ISSUE THREAD

**Issue by Geremia** [Ramblurr/yubi-goog#4]
These scripts only produce the correct OTPs for my Google GMail account. I've tried to use them for Google Authenticator with my other non-Google accounts, but for some reason the OTP codes won't work. Why would that be?

**Comment by Ramblurr** [comment]
Did you program your yubi key with the personalization tool to use the secret key from the other service? You must provide the hex formatted secret, not the base 32 ecoded one (that's what the `--convert-secret` option is for).

I've never tested it with another service, but it should work in theory if the service is using TOTP in the same was as google.

**Comment by Geremia** [comment]
Yes, I used `--convert-secret`. I followed exactly the same procedure that worked for setting up 2FA for my GMail account.

I'm thinking that other services would require `yubi-goog` to communicate with a server or something?
